src/rust/engine/process_executor/src/main.rs

```diff
@@ -297,40 +184,14 @@ async fn main() {
 
     let runner: Box<dyn process_execution::CommandRunner> = match args.server {
         Some(address) => {
-            let root_ca_certs = args
-                .execution_root_ca_cert_file
-                .map(|path| std::fs::read(path).expect("Error reading root CA certs file"));
-
-            let client_certs = args
-                .cas_client_certs_file
-                .as_ref()
-                .map(|path| std::fs::read(path).expect("Error reading root client certs file"));
-
-            let client_key = args.cas_client_key_file.as_ref().map(|path| {
-                std::fs::read(path).expect("Error reading client authentication key file")
-            });
-
-            let mtls_data = match (client_certs, client_key) {
-                (Some(certs), Some(key)) => Some((certs, key)),
-                (None, None) => None,
-                _ => {
-                    panic!(
-                        "Must specify both --cas-client-certs-file and --cas-client-key-file or neither"
-                    )
-                }
-            };
-
-            let tls_config = grpc_util::tls::Config::new(root_ca_certs, mtls_data)
-                .expect("failed parsing root CA certs");
-
-            if let Some(oauth_path) = args.execution_oauth_bearer_token_path {
-                let token = std::fs::read_to_string(oauth_path)
-                    .expect("Error reading oauth bearer token file");
-                headers.insert(
-                    "authorization".to_owned(),
-                    format!("Bearer {}", token.trim()),
-                );
-            }
+            let tls_config = grpc_util::tls::Config::new_from_files(
```

Comment:
This TLS config is slightly different than the CAS one (it uses a different root CA cert file), hence needing a `new_from_files()` on Config, instead of just burying this in the Store initialization code.